sds_data_manager/lambda_code/batch_starter.py

```diff
@@ -441,21 +443,17 @@ def lambda_handler(event: dict, context):
 
         grouped_list = prepare_data(downstream_instruments_to_process)
 
-        # Start Step function execution for each instrument
+        # TODO: more changes required for dates, version, and descriptor
+        # Start Batch Job execution for each instrument
         for instrument_name in grouped_list:
-            input_data = {
-                "command": [
-                    instrument_name,
-                    f"{grouped_list[instrument_name]}",
-                    f"{version}",
-                ]
-            }
-
-            response = step_function_client.start_execution(
-                stateMachineArn=state_machine_arn,
-                name=f"{instrument_name}",
-                input=json.dumps(input_data),
-            )
-            print(
-                f"Started Step Function for {instrument_name} with response: {response}"
-            )
+            for data_level in grouped_list[instrument_name]:
+                batch_client.submit_job(
+                    jobName=f"imap_{instrument_name}_{data_level}_<descriptor>_<startdate>_<enddate>_<vxx-xx>.cdf",
+                    jobQueue=job_queue,
+                    jobDefinition=job_definition,
+                    containerOverrides={
+                        "command": [
+                            f"imap_{instrument_name}_{data_level}_<descriptor>_<startdate>_<enddate>_<vxx-xx>.cdf"
```

Comment:
This is not a comment for this PR but food for thought for later. I was envisioning something like this for future command(of course it's subject to change but rough of what it may look like). 
```
           "command": [
                "python",
                "imap_cli --instrument glows --level l1a --filename 's3://sds-data-449431850278/imap/glows/l0/2023/07/imap_glows_l0_1_20230724_20230724_v02-01.pkts' --dependency {.....}"
            ],
```

Reply:
Agree with you @tech3371, minus the `python` portion which we won't want if we have the cli installed properly, and that one long string has to be separate entries in the command I think.

For now though, I don't know if this command will work or not since it is just a filename. We might want `python run-my-script.py filename.cdf` so you can update the script without needing to update the CDK command.

Reply:
Yep. As long as it fits into the batch job command limits. I'll work on batch_starter.py more in the next PR (hence the todo) and figure out what we can actually pass.

Reply:
yes. Thank you!